### PR TITLE
Wdp 190701 32

### PR DESCRIPTION
**Pull request by dwabe**
New section (Client Feedback) added. Now our customer will be able to add comments of his clients at the bottom of the website.

### CODE REVIEW

File: src/sass/components/_section--feedback.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
+.section--feedback {
+  margin-bottom: 60px;
+
+  .panel-bar {
+    margin-bottom: 30px;
+
+    .row > * {
+      border-bottom: 4px solid #e2e2e2;
```
Comment (Varenthein) [inline src/sass/components/_section--feedback.scss:8]: Using `*` is not the best option. It's a quite inefficient option. Use other selector. You may find proepr elements by className for instance.

File: src/sass/components/_section--feedback.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    .heading {
+      position: relative;
+
+      h3 {
+        color: $primary;
```
Comment (Varenthein) [inline src/sass/components/_section--feedback.scss:15]: We shouldn't style by heading tags. Add className for this heading and use it here as selector.

**Review by Varenthein — CHANGES_REQUESTED** [review]

**Comment by Varenthein** [comment]
And another thing – please use correct PR name.

File: src/sass/components/_section--feedback.scss (hunk 1 above)
Comment (dwabe) [reply src/sass/components/_section--feedback.scss:8]: OK no problem :)

File: src/sass/components/_section--feedback.scss (hunk 2 above)
Comment (dwabe) [reply src/sass/components/_section--feedback.scss:15]: OK no problem :)

**Comment by dwabe** [comment]
Sure - i'll change the name of this PR. I don't know why I decided to change it from default this particular time :)

File: src/sass/components/_section--feedback.scss (hunk 2 above)
Comment (dwabe) [reply src/sass/components/_section--feedback.scss:15]: done, but this first solution is used everywhere - in sections/components. shouldn't we change it?

File: src/sass/components/_section--feedback.scss (hunk 1 above)
Comment (dwabe) [reply src/sass/components/_section--feedback.scss:8]: done, but this first solution is used everywhere - in sections/components. shouldn't we change it?

**Comment by dwabe** [comment]
Requested changes added.

File: src/sass/components/_section--feedback.scss (hunk 2 above)
Comment (Varenthein) [reply src/sass/components/_section--feedback.scss:15]: Yes, it would be a good thing :) We'll add new task for this.

**Pull request merged**